### DISCUSSION TITLE
How to allow node' setImmediate?

### DISCUSSION THREAD

**Discussion by DoneDeal0** [webpack/webpack#12525] [Q&A]
How to allow node' setImmediate in Webpack 5? In webpack 4 I could set this property to true in the node object, but the in Webpack 5 the build crashes, claiming that the configuration object that does not match the API schema.

My code is:

```javascript
  node: {
      global: true,
      __filename: false,
      __dirname: false
     // setImmediate: true (here commented, because it crashes otherwise)
    },
```

**Comment by alexander-akait** [comment]
`globalThis.setImmediate = yourCode` in your code at the first lines of entry

**Reply by DoneDeal0** [reply]
Could you please provide an exemple? I don't see where to write this in my webpack config.

**Reply by alexander-akait** [reply]
You should not write this code inside `webpack.config.js`, you should put this code inside your entry(entries)
